src/assets/js/fluent.js

```diff
@@ -0,0 +1,79 @@
+import { FluentBundle, FluentResource } from '@fluent/bundle';
+import insane from 'insane';
+import Strings from '@/locales/en-US/strings.ftl';
+
+class Fluent {
+  constructor(locale = 'en-US') {
+    const resource = new FluentResource(Strings);
+    const bundle = new FluentBundle(locale);
+    const errors = bundle.addResource(resource);
+    if (errors.length) {
+      console.error(errors);
+    }
+
+    this.bundle = bundle;
+  }
+
+  get(...parameters) {
+    if (typeof parameters[0] === 'string') {
+      switch (parameters.length) {
+        case 2:
+          if (typeof parameters[1] === 'string') {
+            return this.get({ id: parameters[0], attr: parameters[1] });
+          }
+          return this.get({ id: parameters[0], args: parameters[1] });
+        default:
+          return this.get({ id: parameters[0] });
+      }
+    }
+    const { id, attr, args, tags } = parameters[0];
+    const message = this.getMessage(id, attr, args);
+    return Fluent.sanitize(message, tags);
+  }
+
+  static sanitize(message, tags = {}) {
+    return insane(message, {
+      filter(token) {
+        const name = token.attrs['data-l10n-name'];
+        if (Object.keys(tags).includes(name) && tags[name].tag === token.tag) {
+          Object.keys(token.attrs).forEach((k) => {
+            if (k !== 'title') delete token.attrs[k];
+          });
+          Object.entries(tags[name]).forEach(([k, v]) => {
```

Comment:
Not sure how many items this might have, but as a point of reference, I usually go with for loop since it's the faster iterator: https://hackernoon.com/javascript-performance-test-for-vs-for-each-vs-map-reduce-filter-find-32c1113f19d7

Not that it's that big of a deal necessarily depending on how many tags you'll have but, just a thought.

Reply:
Oh man I've now read way too many articles on for vs forEach vs for...in vs for...of and I think I'm more confused than when I started!

I like for...of (readability wise it's the same as forEach, at least in my eyes), but our linter currently prohibits it: `iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations (no-restricted-syntax)`